### PR TITLE
Add JS Interactive 2018 banner

### LINKED ISSUE #1730: Addin JS Interactive to website homepage?
I wanted to see if it would be possible to add a link to JS Interactive to the website? Either as a banner as we've done with the security warning or as a potential tab that links back to the JS Interactive website - see what JS Foundation has done: https://js.foundation.

We've done this in the past with Node.js Interactives and its driven a lot of traffic to our event websites. Please let me know if this is okay with the community. If so, I would like to share a special UTM link. Thanks for your time and look forward to hearing back from the group on this.

### PR DESCRIPTION
**Pull request by fhemberger**
Closes #1730

/cc @ZibbyKeaton

### CODE REVIEW

File: locale/en/blog/uncategorized/profiling-node-js.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ It's incredibly easy to visualize where your Node program spends its time using
 
 You'll be looking at something like this:
 
-<a href="https://www.cs.brown.edu/~dap/helloworld.svg" class="imagelink"><img src="https://www.cs.brown.edu/~dap/helloworld.svg" alt="'Hello World' HTTP server flame graph" /></a>
+<a href="http://cs.brown.edu/people/dapachec/helloworld.svg" class="imagelink"><img src="http://cs.brown.edu/people/dapachec/helloworld.svg" alt="'Hello World' HTTP server flame graph" /></a>
```
Comment (lpinca) [inline locale/en/blog/uncategorized/profiling-node-js.md]: This seems an unrelated change. Can we keep `https`? It seems to works fine.

**Comment by fhemberger** [comment]
@lpinca Thanks, fixed!

**Review by lpinca — APPROVED** [review]
